patches/chrome-installer-util-BUILD.gn.patch

```diff
@@ -1,16 +1,22 @@
 diff --git a/chrome/installer/util/BUILD.gn b/chrome/installer/util/BUILD.gn
-index c66a6f6245ebe7d4abb1993c93f279e9e8442993..d78f0f1867d4b863133f826b45116dfe62813489 100644
+index c66a6f6245ebe7d4abb1993c93f279e9e8442993..eb0398139da2aa09472c9179de3320c96333bdff 100644
 --- a/chrome/installer/util/BUILD.gn
 +++ b/chrome/installer/util/BUILD.gn
-@@ -246,9 +246,16 @@ action("generate_strings") {
+@@ -246,9 +246,22 @@ action("generate_strings") {
      "$target_gen_dir/installer_util_strings.rc",
    ]
  
 +  brand = "$branding_path_component"
```

Comment:
Why are we changing the branding_path_component here?

Reply:
This change is not introduced in this PR.
https://github.com/brave/brave-browser/issues/153 tracks this.

Reply:
sorry, you're right. I'm going to take a look at that issue